OutOfSchool/OutOfSchool.WebApi/Controllers/V1/AchievementController.cs

```diff
@@ -94,6 +96,7 @@ public async Task<IActionResult> GetByWorkshopId([FromQuery] AchievementsFilter
     [ProducesResponseType(StatusCodes.Status401Unauthorized)]
     [ProducesResponseType(StatusCodes.Status403Forbidden)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    [FeatureGate(nameof(Feature.CreateAchievement))]
```

Comment:
you can also add it for update and delete, can leave get methods without feature gate.